### ISSUE TITLE
Adding Columns

### ISSUE THREAD

**Issue by huwred** [robertjf/umbMemberListView#43]
How can you add columns in the latest version?

if you edit the memberManager.controller.js config section changes just get overwritten when project rebuilt.

**Comment by robertjf** [comment]
At the moment, you can't - I do have plans to make this configurable, however.

**Comment by akeilox** [comment]
Hi Robert!
Would that feature be available in version 7 by any chance? I am maintaining latest v7 installation with MemberListView on it.

I could swear I asked this question and fiddled with adding a column Member List View before few years ago but I cant find/recall .....

**Comment by robertjf** [comment]
Hey @akeilox it's unlikely, given that Umbraco 7 is all but unsupported - we'll more likely be targeting Umbraco 8 and the latest LTS and STS versions when we do.

At this point, I've not had any time spend on this, so there's no projected date for this yet either.

**Comment by akeilox** [comment]
Thanks for the reply Robert. I see our discussion from 2014 in the umbraco packages' issues-pages. Time flies!

Do you recall by any chance if custom member fields (that gets indexed by examine) can be shown by hacking the code in v7?

**Comment by robertjf** [comment]
yes they can - take a look at the source (angular) and you should be able to work it out
